addons/mail/static/src/components/file_uploader/tests/file_uploader_tests.js

```diff
@@ -20,7 +20,7 @@ QUnit.module('file_uploader_tests.js', {
         this.components = [];
 
         this.createFileUploaderComponent = async props => {
-            return createRootMessagingComponent(this, "FileUploader", {
+            createRootMessagingComponent(this, "FileUploader", {
```

Comment:
Should have await instead of return.

Also commit title is wrong, this commit is not about component ref.